packages/skill-lesson/markdown/mdx.tsx

```diff
@@ -1,4 +1,5 @@
-import * as React from 'react'
+'use client'
```

Comment:
this could probably be moved to a "closer" component wrapper, but had to be used to get this to render

if we were using RSC for the actual presentation and not the just the "backend" this might affect seo